product_docs/docs/jdbc_connector/42.7.3.2/01_jdbc_rel_notes/jdbc_42.7.3.2_rel_notes.mdx

```diff
@@ -0,0 +1,33 @@
+---
+title: "EDB JDBC Connector 42.7.3.2 release notes"
+navTitle: Version 42.7.3.2
+---
+
+Released: 20 Nov 2024
+
+The EDB JDBC connector provides connectivity between a Java application and an EDB Postgres Advanced Server database.
+
+New features, enhancements, bug fixes, and other changes in the EDB JDBC Connector 42.7.3.2 include:
+
+| Type                | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            
+|---------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Performance         | Improved parsing performance with large SQL (MTK/SQL Plus)                                                                                                                                                                                                                        |                                                                                                                                                                                                                                                           
+| Enhancement         | Added support for EDB Postgres Advanced Server 17.2                                                                                                                                                                                                                               |                                                                                                                                                                                                                                                                                                                                                                          
+| Fix                 | The JDK 8 compatibility is broken in ver 42.7.3.1 of edb-jdbc18 driver.                                                                                                                                                                                                           |                                                   
```

Comment:
Usually, we distinguish them between security fixes or bug fixes. I would suggest going over the list and checking which could be bugs or security to categorize them that way? 

Also, there is a style/way in which we phrase the notes. The [Release Notes Style Guide](https://www.enterprisedb.com/docs/community/contributing/releasenotes-styleguide/) has some examples. I am changing this bullet here as I would to it, but I leave it up to you whether you want to do the adaptations that way or not :) 
```suggestion
| Bug fix                 | Fixes an incompatibility issue with JDK 8 that found in version 42.7.3.1 of the `edb-jdbc18` driver.                                                                                                                                                                                                           |                                                   
```